docs/DeveloperGuide.md

```diff
@@ -329,26 +329,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case ends.
 
-**Use case: Saving data**
+**UC3 - Saving data**
 
 **MSS**
 
-1.  User enters a valid command that alters data (`client add`, `client delete`, `client note add`, etc).
+1.  User enters a valid command that alters data (eg <ins>`adding a client (UC1)`</ins>).
```

Comment:
Minor point, but can this be changed to "E.g." instead of "eg" with the period.

Reply:
Just to add on, I don't think <ins> tag should be used here, as markup defines it as inserting something? e.g. [here](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_del_css). <u> seems more appropriate if the intention is to underline the text.

Reply:
Okay I have made the changes, can you guys check again?

Reply:
LGTM